app/controllers/dashboards_controller.rb

```diff
@@ -3,7 +3,6 @@ class DashboardsController < BaseController
   before_action :set_policy_with_context, except: [:refresh_dashboard]
   before_action :set_dashboard
   before_action :set_defence_requests, only: [:show, :closed]
-  before_action :set_dashboard_count, only: [:show, :closed]
```

Comment:
Ah, I see. So, making it a singular resource seems unrelated to that - there is nothing that stops that happening or being Restful. You'd be showing a dashboard. Regarding the scoping: we should probably take all the necessary measures to only call scopes on `DefenceRequest` within a policy. Anything after that should just traverse the set returned. 

I like the notion of passing an `id` in order to have the open/closed dashboard on a different url, for instance, though.